docs/plugins/source.rst

```diff
@@ -3,11 +3,11 @@
 Source Plugins
 ==============
 
-Source plugins handle artifact import. All source plugins maintain state
-between runs, allowing them to skip previously processed tweets, blog posts,
-etc and get right to work finding new indicators.
+Source plugins handle artifact import. Artifacts include tweets, blog posts, etc.
```

Comment:
Artifacts are URLs, IPAddress, YARASignature, etc - the artifacts are extracted from tweets, blog posts, etc as defined by the `sources`. I didn't word this part very well, any ideas how to improve it?

Reply:
How about 
`For each source specified, ``ThreatIngestor`` imports artifacts, which can be URLs, IP Addresses, YARA Signatures, etc.   `

And maybe elaborate in the beginning that the sources may link to tweets, blog posts, etc.

Reply:
That sounds good.